utils/urlParams/index.ts

```diff
@@ -1,22 +1,25 @@
+import { ParsedUrlQuery } from 'querystring'
```

Comment:
What library is this? On npm it says to use URLSearch params https://www.npmjs.com/package/querystring

Reply:
Hmm yeah that was an attempt to please TypeScript.

The Next.js router uses ParsedUrlQuery internally, which is basically a `Record<string, string | string[]>`, while the `URLSearchParams` type is `Record<string, string>`. We need to interpret the former as the latter somewhere, but prefer not to do it each time we pass `router.query` to `navigate`.

Not sure if this even works properly with array type query params, as the format from `router.query` does not seem completely compatible with `URLSearchParams`.

Reply:
This would require testing, but furthermore, this util seems recurrent in all of the repos, so fixing something in only one of the apps does not seem too great. Maybe it should also move to shared components